### PR TITLE
task5648 edit post

### PR DESCRIPTION
**Pull request by laam55**
https://edu-redmine.framgia.vn/issues/5648

### CODE REVIEW

File: app/Http/Requests/EditPostRequest.php [hunk 1]
```diff
@@ -23,20 +24,30 @@ public function authorize()
...
     {
+        $postId = $this->route('id');
+        $tagMinRule = new TagMinRule(config('constants.MIN_TAG'));
+
         return [
-            'title' => 'required|min:6|unique:posts,title,' . $this->id,
-            'content' => 'required|min:6',
+            'title' => 'required|min:6|max:150|unique:posts,title,' . $postId,
+            'content' => 'required|max:5000',
+            'tags' => [
```
Comment (framgiaciv3) [inline app/Http/Requests/EditPostRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/EditPostRequest.php</br>**Line**: 33</br>Each line in an array declaration must end in a comma

File: app/Http/Requests/EditPostRequest.php [hunk 2]
```diff
@@ -23,20 +24,30 @@ public function authorize()
...
+                'required',
+                'min' => $tagMinRule,
```
Comment (framgiaciv3) [inline app/Http/Requests/EditPostRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.KeySpecified</br>**Severity**: error</br>**File**: app/Http/Requests/EditPostRequest.php</br>**Line**: 35</br>Key specified for array entry; first entry has no key

File: app/Repositories/Eloquents/PostRepository.php [hunk 3]
```diff
@@ -301,4 +301,17 @@ public function createPostsTags($tags, $postId)
 
         return true;
     }
+
+    public function updatePostsTags($tags, $post_id)
```
Comment (framgiaciv3) [inline app/Repositories/Eloquents/PostRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PostRepository.php</br>**Line**: 305</br>Variable "post_id" is not in valid camel caps format

File: app/Repositories/Eloquents/PostRepository.php [hunk 4]
```diff
@@ -301,4 +301,17 @@ public function createPostsTags($tags, $postId)
...
+    {
+        $post = $this->model()->find($post_id);
```
Comment (framgiaciv3) [inline app/Repositories/Eloquents/PostRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PostRepository.php</br>**Line**: 307</br>Variable "post_id" is not in valid camel caps format

File: app/Http/Requests/EditPostRequest.php (hunk 1 above)
Comment (framgiaciv3) [inline app/Http/Requests/EditPostRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/EditPostRequest.php</br>**Line**: 33</br>Each line in an array declaration must end in a comma

**Comment by laam55** [comment]
ready

**Pull request merged**